radiation/python/radlw/setcoef.py

```diff
@@ -256,6 +256,7 @@ def setcoef(pavel, tavel, tz, stemp, h2ovmr, colamt, coldry, colbrd,
 
 serializer = ser.Serializer(ser.OpenModeKind.Read, ddir, "Serialized_rank0")
 savepoints = serializer.savepoint_list()
+print(savepoints[10])
```

Comment:
do you need this?

Reply:
Nope. Fixed, and also changed reading from serialized data to reference by name rather than array index